pkg/workload/workload.go

```diff
@@ -51,6 +51,12 @@ func NewInfo(w *kueue.Workload) *Info {
 	}
 }
 
+func (i *Info) Update(wl *kueue.Workload, cqName string) {
+	i.Obj = wl
+	i.TotalRequests = totalRequests(&wl.Spec)
+	i.ClusterQueue = cqName
```

Comment:
no need to populate clusterQueue here. It's populated here https://github.com/kubernetes-sigs/kueue/blob/68ae767b6da43ff376e4e381739cc8152f0da884/pkg/queue/manager.go#L481, only when scheduling.